### PR TITLE
feat: Change table margin for Tx Hash

### PR DESCRIPTION
**Pull request by jinoosss**

### CODE REVIEW

File: src/pages/transactions/[...hash].tsx
Comment (r3v4s) [inline src/pages/transactions/[...hash].tsx:1]: what for?

File: src/repositories/api/fetchers/api-transaction-details.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import axios, {AxiosResponse} from 'axios';
 import {API_URI, API_VERSION} from '@/common/values/constant-value';
 
 export const getTransactionDetails = async (
-  hash: string | any,
+  hash: string[] | any,
```
Comment (r3v4s) [inline src/repositories/api/fetchers/api-transaction-details.ts:5]: does array stands from multi-msg ?

**Review by r3v4s — APPROVED** [review]

File: src/pages/transactions/[...hash].tsx
Comment (jinoosss) [reply src/pages/transactions/[...hash].tsx:1]: The transaction hash contains a slash character.
This processing gets all the subpaths.

File: src/repositories/api/fetchers/api-transaction-details.ts (hunk 1 above)
Comment (jinoosss) [reply src/repositories/api/fetchers/api-transaction-details.ts:5]: Recombines slash-separated subcharacters.

File: src/pages/transactions/[...hash].tsx
Comment (r3v4s) [reply src/pages/transactions/[...hash].tsx:1]: ha.. encoding problem :(

**Pull request merged**